src/FormatNumber/Parser.elm

```diff
@@ -211,6 +214,67 @@ getDecimals locale digits =
             addZerosToFit min digits
 
 
+
+{- Joins parts into `Maybe Float`. It expects parts to be strings with only digits.
```

Comment:
I may be wrong, but I think the style suggests wrapping at 80 chars.

```suggestion
{- Joins parts into `Maybe Float`. It expects parts to be strings with only
digits.
```

Reply:
ok. I will make this change